### PR TITLE
refactor(api): use process env instead of config service

### PR DESCRIPTION
**Pull request by eatmyvenom**

### CODE REVIEW

**Review by ugcodrr — APPROVED** [review]

File: apps/api/src/app.module.ts [hunk 1]
```diff
@@ -14,24 +13,11 @@ import { SkinModule } from './skin';
...
-    }),
-    RedisModule.forRootAsync({
-      imports: [ConfigModule],
-      useFactory: (configService: ConfigService) => ({
-        config: {
-          url: configService.get<string>('REDIS_URL', ''),
-        },
-      }),
-      inject: [ConfigService],
+    TypegooseModule.forRoot(process.env.MONGODB_URI as string, {}),
```
Comment (jacobk999) [inline apps/api/src/app.module.ts:16]: add `MONGODB_URI` to `module.d.ts`

File: apps/api/src/app.module.ts [hunk 2]
```diff
@@ -14,24 +13,11 @@ import { SkinModule } from './skin';
...
+    RedisModule.forRoot({
+      config: {
+        url: process.env.REDIS_URL,
```
Comment (jacobk999) [inline apps/api/src/app.module.ts:19]: add `REDIS_URL` to `module.d.ts`

File: apps/api/src/hypixel/hypixel.module.ts [hunk 3]
```diff
@@ -1,20 +1,15 @@
...
-        headers: {
-          'API-Key': configService.get<string>('HYPIXEL_API_KEY', ''),
-        },
-        timeout: configService.get<number>('HYPIXEL_API_TIMEOUT', 5000),
-      }),
-      inject: [ConfigService],
+    HttpModule.register({
+      baseURL: `https://api.hypixel.net/`,
+      headers: {
+        'API-Key': process.env.HYPIXEL_API_KEY as string,
```
Comment (jacobk999) [inline apps/api/src/hypixel/hypixel.module.ts:10]: add `HYPIXEL_API_KEY ` to `module.d.ts`

File: apps/api/src/hypixel/hypixel.module.ts [hunk 4]
```diff
@@ -1,20 +1,15 @@
...
+      },
+      timeout: Number(process.env.HYPIXEL_API_TIMEOUT) ?? 5000,
```
Comment (jacobk999) [inline apps/api/src/hypixel/hypixel.module.ts:12]: add `HYPIXEL_API_TIMEOUT` to `module.d.ts`

**Review by jacobk999 — CHANGES_REQUESTED** [review]

File: apps/api/src/app.module.ts (hunk 1 above)
Comment (jacobk999) [inline apps/api/src/app.module.ts:16]: Remove the cast

File: apps/api/src/hypixel/hypixel.module.ts (hunk 3 above)
Comment (jacobk999) [inline apps/api/src/hypixel/hypixel.module.ts:10]: Remove the cast

**Review by jacobk999 — CHANGES_REQUESTED** [review]

**Pull request merged**